packages/compiler-vapor/__tests__/__snapshots__/compile.spec.ts.snap

```diff
@@ -1,5 +1,17 @@
 // Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
 
+exports[`compile > asset imports > from public directory 1`] = `
+"import { setProp as _setProp, renderEffect as _renderEffect, template as _template } from 'vue';
+import _imports_0 from '/vite.svg';
```

Comment:
```js
import _imports_0 from '/vite.svg';
const t0 = _template(`<img src="${_imports_0}">`, true)
```

Maybe it should be done like this to avoid `renderEffect` and `setProp`.

Reply:
It should be easier to implement using string concatenation. For example, like this:
- Wrap the variable with special characters
```js
template += ` ${key.content}="$\{${values[0].content}\}$"`
```
- In genTemplates, replace the variable with string concatenation
```js
${JSON.stringify(template).replace(/\${_imports_(\d+)}\$/g,'"+ _imports_$1 +"')
```